### PR TITLE
Merge branch 'main' of https://github.com/ajaynegi45/Uttarakhand-Culture-NewUI

### PR DESCRIPTION
**Pull request by Mallik-vinukonda**
https://github.com/user-attachments/assets/9e3b7c30-cf91-409a-b418-c2929c4e1a98

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,11 +14,15 @@
     "@neondatabase/serverless": "^0.9.5",
     "@vercel/speed-insights": "^1.0.12",
     "bcryptjs": "^2.4.3",
+    "dotenv": "^16.4.5",
```
Comment (ajaynegi45) [inline package.json:17]: Remove it

File: package.json [hunk 2]
```diff
@@ -14,11 +14,15 @@
...
     "drizzle-kit": "^0.24.2",
     "drizzle-orm": "^0.33.0",
-    "next": "14.2.5",
+    "framer-motion": "^11.11.10",
+    "gsap": "^3.12.5",
+    "next": "^14.2.16",
     "next-auth": "^5.0.0-beta.22",
     "nodemailer": "^6.9.15",
+    "pg": "^8.13.0",
```
Comment (ajaynegi45) [inline package.json:25]: remove it

File: package.json [hunk 3]
```diff
@@ -14,11 +14,15 @@
     "@neondatabase/serverless": "^0.9.5",
     "@vercel/speed-insights": "^1.0.12",
     "bcryptjs": "^2.4.3",
+    "dotenv": "^16.4.5",
     "drizzle-kit": "^0.24.2",
     "drizzle-orm": "^0.33.0",
-    "next": "14.2.5",
+    "framer-motion": "^11.11.10",
+    "gsap": "^3.12.5",
```
Comment (ajaynegi45) [inline package.json:21]: framer-motion and gsap use only framer.

File: src/app/explore/page.module.css [hunk 4]
```diff
@@ -1,77 +1,73 @@
...
-    align-items: center;
-    margin: 3rem 0 5rem 0;
-    color: #ffff;
+.pageHeader {
+	display: flex;
+	flex-direction: column-reverse;
+	justify-content: center;
+	align-items: center;
+	margin: 3rem 0 5rem 0;
+	color: #ffff;
```
Comment (ajaynegi45) [inline src/app/explore/page.module.css:7]: Why you change the spacing. Who said you to change it. Make it as it was earlier

**Review by ajaynegi45 — CHANGES_REQUESTED** [review]
Hi @Mallik-vinukonda,

Thank you for your prompt work on this issue. I appreciate your dedication to the project.

I have reviewed your work, and I need some changes.


94:6  Error: 'Card' is not defined.  react/jsx-no-undef
103:6  Error: 'Card' is not defined.  react/jsx-no-undef
112:6  Error: 'Card' is not defined.  react/jsx-no-undef
121:6  Error: 'Card' is not defined.  react/jsx-no-undef
130:6  Error: 'Card' is not defined.  react/jsx-no-undef
139:6  Error: 'Card' is not defined.  react/jsx-no-undef
148:6  Error: 'Card' is not defined.  react/jsx-no-undef
157:6  Error: 'Card' is not defined.  react/jsx-no-undef
166:6  Error: 'Card' is not defined.  react/jsx-no-undef
./src/components/about/About.tsx
21:9  Warning: Using `` could result in slower LCP and higher bandwidth. Consider using `<Image />` from `next/image` to automatically optimize images. This may incur additional usage or cost from your provider. See: https://nextjs.org/docs/messages/no-img-element  @next/next/no-img-element
48:9  Warning: Using `` could result in slower LCP and higher bandwidth. Consider using `<Image />` from `next/image` to automatically optimize images. This may incur additional usage or cost from your provider. See: https://nextjs.org/docs/messages/no-img-element  @next/next/no-img-element
./src/components/ui/TrekCard.tsx
125:9  Warning: Using `` could result in slower LCP and higher bandwidth. Consider using `<Image />` from `next/image` to automatically optimize images. This may incur additional usage or cost from your provider. See: https://nextjs.org/docs/messages/no-img-element  @next/next/no-img-element
info  - Need to disable some ESLint rules? Learn more here: https://nextjs.org/docs/basic-features/eslint#disabling-rules
Error: Command "npm run build" exited with 1


Once again, thank you for your contribution! ❤️

File: package.json (hunk 3 above)
Comment (Mallik-vinukonda) [reply package.json:21]: > framer-motion and gsap use only framer.

Ok i will

File: src/app/explore/page.module.css (hunk 4 above)
Comment (Mallik-vinukonda) [reply src/app/explore/page.module.css:7]: > Why you change the spacing. Who said you to change it. Make it as it was earlier

ok , that's my bad , i apoloize for that

File: package.json (hunk 2 above)
Comment (Mallik-vinukonda) [reply package.json:25]: > remove it

i got errors while setting up the project , so i did it

Comment (Mallik-vinukonda) [reply package.json:25]: > > remove it
> 
> i got errors while setting up the project , so i did it

Error: No database connection string was provided to `neon()`. Perhaps an environment variable has not been set?

This error happened while generating the page. Any console logs will be displayed in the terminal window.
Source
src/lib/drizzle.ts (8:19) @ connectionString

   6 | const connectionString = process.env.DATABASE_URL!;
   7 |
>  8 | const pool = neon(connectionString)
     |                   ^
   9 |
  10 | export const db = drizzle(pool, { schema })
Call Stack

Comment (Mallik-vinukonda) [reply package.json:25]: > > remove it
> 
> i got errors while setting up the project , so i did it

Error: No database connection string was provided to `neon()`. Perhaps an environment variable has not been set?

This error happened while generating the page. Any console logs will be displayed in the terminal window.
Source
src/lib/drizzle.ts (8:19) @ connectionString

   6 | const connectionString = process.env.DATABASE_URL!;
   7 |
>  8 | const pool = neon(connectionString)
     |                   ^
   9 |
  10 | export const db = drizzle(pool, { schema })
Call Stack

**Pull request closed without merging**